exporters/jaeger/build.gradle.kts

```diff
@@ -42,8 +42,8 @@ wire {
   }
 }
 
-tasks {
-  compileJava {
-    source("$buildDir/generated/source/wire")
+sourceSets {
+  main {
+    java.srcDir("$buildDir/generated/source/wire")
```

Comment:
Without this my IDE doesn't pick up the generated proto classes.

Reply:
Hmm - somehow it seems OK for me but maybe I'm sporadically in a good state. sourceSets does seem better

Can you make the same change to https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/otlp/common/build.gradle.kts#L72 ?

Reply:
Done.